guide/daily-spending-wallet/security.md

```diff
@@ -362,6 +371,18 @@ By including UX patterns for hiding information pattern wallets, we give users a
 
 ---
 
+## Preventing theft when offline
+
+Mobile wallets may frequently be offline, either due to a poor connection or because the application is not running. To improve battery live, mobile operating systems tend to prevent applications the user is not actively using from running code in the background. This can prevent mobile wallets from being reliable monitors of its payment channels. Malicious actors who share channels with the user may take advantage of these moments to try to steal funds by closing channels with incorrect data.
```

Comment:
```suggestion
Lightning wallets need to be online to monitor their payment channels to prevent their counter parties from attempting to steal their bitcoin. Mobile wallets however are frequently offline. This is usually due to a poor connection or the user not having the wallet open on their device. 

Most major mobile operating systems do not allow apps open in the background to run tasks, such as monitoring a lightning node. This is to improve battery life as well as prevent apps acting maliciously in the background. 

This prevents mobile lightning wallets from being reliable monitors of its payment channels. Malicious actors could take advantage of this and attempt to steal the mobile wallet users bitcoin by closing channels with incorrect data.
```

Stated the problem from the start and some other copy / readability changes.